src/analyze/report.ts

```diff
@@ -77,19 +79,17 @@ export async function report(options: Options) {
         ...result.stats,
         extraStats
       };
-      if (result.stats.extraStats) {
+      if (result.stats.extraStats?.length) {
```

Comment:
the check was probably fine before since the `for` would be no-op over a 0-length array